### PR TITLE
Info regarding portrait box, live actor direction and music in dungeon

### PR DESCRIPTION
**Pull request by marius851000**
I’m still a bit unsure in which order I want to add stuff in pmdsky-debug. I think I’ll for now focus in the palette in overlay11 stuff. But I did that in the meantime.

Also, I think portrait_box is the "kaomodo" struct in this PR: https://github.com/UsernameFodder/pmdsky-debug/pull/21/files

### CODE REVIEW

File: headers/types/common/common.h [hunk 1]
```diff
@@ -144,20 +144,13 @@ struct dialog_box {
 };
 ASSERT_SIZE(struct dialog_box, 224);
 
-// Structure for dialog boxes with portraits?
+// Represent a portrait that appear inside a dialogue box
 struct portrait_box {
```
Comment (UsernameFodder) [inline headers/types/common/common.h:148]: > Also, I think portrait_box is the "kaomodo" struct in this PR: https://github.com/UsernameFodder/pmdsky-debug/pull/21/files

Hmm I think you're very likely correct. Could we maybe copy the fields over from https://github.com/UsernameFodder/pmdsky-debug/pull/21 (with comments saying that they need verification)?

**Review by UsernameFodder — CHANGES_REQUESTED** [review]

**Comment by UsernameFodder** [comment]
(The formatting error is should go away once you rebase this branch against `master` and rerun the formatter)

File: headers/types/common/common.h (hunk 1 above)
Comment (UsernameFodder) [reply headers/types/common/common.h:148]: Actually, on second thought, copying over just that stuff from #21 probably doesn't make sense out of context. We can always port that stuff another day, or try to get the original PR merged.

File: headers/types/common/enums.h [hunk 2]
```diff
@@ -708,6 +708,35 @@ enum monster_id {
 ENUM_16_BIT(monster_id);
 #pragma pack(pop)
 
+// Emotion for portraits
+enum face_emotion {
+    FACE_NORMAL = 0,
```
[the remark this answers is not in the document]
Comment (marius851000) [reply headers/types/common/enums.h:713]: The reason I used FACE is cause it’s what SkyTemple use, but I guess EMOTION_ will do just fine too.

File: headers/types/common/common.h (hunk 1 above)
Comment (marius851000) [reply headers/types/common/common.h:148]: I agree with that (thought kaomodo may not be an appropriate name for this struct, given that’s also how the file format is called)

**Review by UsernameFodder — APPROVED** [review]
Thanks! This portrait stuff is pretty useful info to have.

**Pull request merged**